base/sort.jl

```diff
@@ -689,12 +693,14 @@ function sortperm(v::AbstractVector;
     for (i,ind) in zip(eachindex(p), indices(v, 1))
         p[i] = ind
     end
-    sort!(p, alg, Perm(ordr,v))
+    sort!(p, alg, Perm(order, v))
 end
 
 
 """
-    sortperm!(ix, v; alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward, initialized::Bool=false)
+    sortperm!(ix, v;
+              alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity,
```

Comment:
Not sure what's the question, but it looks exactly the same at the REPL (and in blue):
```
help?> sortperm!
search: sortperm! sortperm

  sortperm!(ix, v;
            alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity,
            rev::Bool=false, initialized::Bool=false)

```